website/blog/2025-12-16-aks-config-scheduler-profiles-preview/index.md

```diff
@@ -0,0 +1,234 @@
+          - name: NodeResourcesFit
+            args:
+              apiVersion: kubescheduler.config.k8s.io/v1
+              kind: NodeResourcesFitArgs
+              scoringStrategy:
+                type: RequestedToCapacityRatio
+                resources:
+                  - name: memory
+                    weight: 5
+                  - name: cpu
+                    weight: 1
+                  - name: ephemeral-storage
+                    weight: 1
+                requestedToCapacityRatio:
+                  shape:
+                    - utilization: 0
+                      score: 0
+                    - utilization: 30
+                      score: 4
+                    - utilization: 60
+                      score: 8
+                    - utilization: 80
+                      score: 10
+                    - utilization: 90
+                      score: 6
+                    - utilization: 100
+                      score: 0
+```
+
+## Best Practices and Configuration Considerations
+
+As a reminder, there are many parameters the scheduler considers across the [scheduling cycle][scheduling-framework/#interfaces] before a pod is placed on a node that impacts how a pod is assigned. This section is meant to help guide how you consider both individual plugin configurations, your custom scheduler configuration, and your Deployment design holistically.
+
+1. Ensure the intended deployment is assigned to the _correct_ scheduler profile.
+2. Ensure the custom scheduler profile complements the implementation of Deployments, StorageClasses, and PersistentVolumeClaim's. Misalignment can lead to pending pods and degraded workload performance, even when the scheduler is functioning as expected.
+3. Ensure there are enough nodes in each zone to accommodate your deployment replicas and ensure your AKS node pool spans the right availability zones. If not, pods may remain in a pending state.
+4. Use namespaces to separate workloads which improves your ability to validate or troubleshoot.
+5. Assign `priorityClassName` for workloads that should preempt others, this is critical if you use the DefaultPreemption plugin.
+6. If you use the `ImageLocality` plugin, use DaemonSets or node pre-pulling for latency-sensitive images, otherwise the benefit may be minimal.
+7. If your cluster is large, a low `PercentageOfNodesToScore` speeds scheduling by reducing the number of nodes scored, _but_ it may reduce optimal placement.
+8. If you enable a plugin in the `plugins:multipoint` section but do not define it in `pluginConfig`, AKS uses the default configuration for that plugin.
+9. For `NodeResourcesFit`, the ratio matters more than absolute values. So CPU:Memory:Storage = 3:1:2, which means CPU is 3× more influential than memory, and storage is 2x more influential than memory in the scoring phase.
+10. Pair `PodTopologySpread` with Pod Disruption Budget's (PDB) and multi‑replica strategies for HA during upgrades.
```

Comment:
These while potentially good best practices, I think they're better suited to docs. They look too random here.

Reply:
The blog post is about an announcement, and it starts at a high level with what scheduling is and what this feature is trying to accomplish, then near the end (here) is talking about best practices. I'm not sure the same reader, assumed here as someone who's not very familiar with this feature would understand what those best practices are or how to apply them from a blog post. And when they do need to apply them, I'd rather they find them out in docs or in a dedicated post on best practices.